pyphi/subsystem.py

```diff
@@ -207,7 +209,8 @@ def cut_node_labels(self):
     @property
     def tpm_size(self):
         """int: The number of nodes in the TPM."""
-        return self.tpm.shape[-1]
+        # forward and backward tpm sizes should be the same
+        return self.forward_tpm.shape[-1]
```

Comment:
I'd add a "defensive programming" check here to make sure they're actually the same, just in case, since this isn't a performance-critical property